README.md

```diff
@@ -4,79 +4,202 @@
 # PyCon Taiwan Blog
 
 ## Getting Started
+### Environment Setup
 
-### Prerequisites
-* [Python 3.10](https://www.python.org/downloads/)
-* [pipenv](https://pipenv.pypa.io/en/latest/)
+1. Fork & Clone [pycontw-blog](https://github.com/pycontw/pycontw-blog)
 
+    For those who already know how to fork the repository, you can skip to the next step.
 
-### Environment Setup
+    <details>
```

Comment:
I would suggest we remove this `details` block. Might not be strictly required

Reply:
My intention is to keep the layout more clean, and try to keep content stay as short as possible.You can choose to fold the section only when you really need to. Anyway, if it's too complex for readers to read the content, I could remove all the `details` tag.

Reply:
In that case, I think I'm ok with this but I would suggest changing `click me` to something more meaning to that specific block